hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -160,7 +160,7 @@ public class HiveSyncConfig extends HoodieSyncConfig {
 
   public static final ConfigProperty<String> HIVE_SUPPORT_TIMESTAMP_TYPE = ConfigProperty
       .key("hoodie.datasource.hive_sync.support_timestamp")
-      .defaultValue("false")
+      .defaultValue("true")
       .withDocumentation("‘INT64’ with original type TIMESTAMP_MICROS is converted to hive ‘timestamp’ type. "
           + "Disabled by default for backward compatibility.");
```

Comment:
This comment is incorrect with this change, as is the one in the FlinkStreamerConfig

Reply:
@lucasmo If hudi can support hive to read the correct timestamp type, this configuration will be removed later. The plan here is to adapt step by step, so my consideration is to change it to true first.

Reply:
@lucasmo Sorry I misunderstood what you mean. I've revised this later ~